README.md

```diff
@@ -51,7 +55,9 @@ rtmouseをインストールして不具合が出た場合のみ以下の設定
 
 * Device Tree機能を「切」にする。
 
-### arm64版Ubuntu18.04
+### for arm64 Ubuntu18.04
+
+Add a following new line in `/boot/firmware/config.txt` to change the i2c_baudrate to 62.5 kHz.
```

Comment:
It may be necessary to set the I2C baudrate lower than the default value ([issues#13](https://github.com/rt-net/RaspberryPiMouse/issues/13)).

みたいな1行がほしいです。（下げなくても動く場合があるらしいので）